src/Model/Profile.php

```diff
@@ -221,7 +221,7 @@ public static function formatLocation(array $profile): string
 	public static function load(App $a, string $nickname, bool $show_contacts = true)
 	{
 		$profile = User::getOwnerDataByNick($nickname);
-		if (empty($profile) || $profile['account_removed']) {
+		if (empty($profile) || !isset($profile['account_removed']) || $profile['account_removed']) {
```

Comment:
Why not `!empty($profile['account_removed'])`?

Reply:
No, the condition works if the key is missing OR trueish, so it can’t be simplified this way. I took the same mental path yesterday but it was a dead end.

Reply:
I realize now the best we could have done is
```suggestion
		if (!isset($profile['account_removed']) || $profile['account_removed']) {
```

Reply:
I'll make include that change into my next PR.

Reply:
I found a similar one in `mod/photos.php` and also made it the way @MrPetovan suggested.